libs/data-mapper/src/lib/models/Function.ts

```diff
@@ -6,6 +6,11 @@ export interface FunctionManifest {
   transformFunctions: FunctionData[];
 }
 
+export enum FunctionType {
+  PseudoFunction = 'PseudoFunction',
+  TransformationFunction = 'TransformationFunction',
```

Comment:
Did you pull the manifest and check to see if there are other ones of these?

Reply:
I did not, lemme do that just to confirm

Reply:
Yeah, all the ones that come from the manifest are "TransformationFunction", so all good there.

Side note, that makes me wonder what its purpose is in the first place :P